Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:lts-alpine AS builder
```

Comment:
I use a tool called [HadoLint](https://github.com/hadolint/hadolint) in VsCode to lint Dokerfiles. It has 3 warning messages and 1 info message for this file. There is nothing drastic but it is probably worth taking a look. If you can't run the tool yourself let me know and I can drop the messages in here or we can have a call.

Reply:
I managed to install it and make it happy :)
Thank you for the recommendation.